### ISSUE TITLE
Multicolumn layout vs choicegroup

### ISSUE THREAD

**Issue by dikey94** [sdaps/sdaps#129]
Good evening everyone,
I'm trying to create a multi column questionnaire which contains ABCD fields only. Something like answers sheet.  What I have for now:

> \begin{multicols}{2}
>     
>     
>       \begin{choicegroup}{ }
>       		\groupaddchoice{A}
>       		\groupaddchoice{B}
>       		\groupaddchoice{C}
>       		\groupaddchoice{D}
> 
>       		\choiceline{1}
>       		\choiceline{2}
>       		\choiceline{3}
>       		\choiceline{4}
>       		\choiceline{5}
>       		\choiceline{6}
>       		\choiceline{7}
>       		\choiceline{8}
>       		\choiceline{9}
>       		\choiceline{10}
>     	\end{choicegroup}
>     	
>     	\begin{choicegroup}{ }
>       		\groupaddchoice{A}
>       		\groupaddchoice{B}
>       		\groupaddchoice{C}
>       		\groupaddchoice{D}
> 
>       		\choiceline{11}
>       		\choiceline{12}
>       		\choiceline{13}
>       		\choiceline{14}
>       		\choiceline{15}
>       		\choiceline{16}
>       		\choiceline{17}
>       		\choiceline{18}
>       		\choiceline{19}
>       		\choiceline{20}
>     	\end{choicegroup}
>       
>       
>     \end{multicols}

The result is not very satisfying. I attach a screenshot as a preview.
Any recommendations? 

Thanks!
![screenshot from 2018-03-22 19-32-42](https://user-images.githubusercontent.com/14232224/37790799-d048a1a8-2e07-11e8-9fcc-3d4dfac0fab5.png)

**Comment by benzea** [comment]
What is your problem? The right alignment of the question text?

For that, you could add an `\hfill` or similar to the question text. The new class handles that a bit nicer :-/

**Comment by dikey94** [comment]
The right alignment + unnecessary 1.2 and 1.2 headers

**Comment by benzea** [comment]
On Thu, 2018-03-22 at 19:20 +0000, dikey94 wrote:
> The right alignment + unnecessary 1.2 and 1.2 headers

I think the headers disappear if you pass an empty string as the
question. Other than that, it is subsection numbering which can be
overriden using the normal LaTeX/KOMA Script methods.
> 
>   
>   
> 
> 
> 
> 
> 
> {"api_version":"1.0","publisher":{"api_key":"05dde50f1d1a384dd78767c5
> 5493e4bb","name":"GitHub"},"entity":{"external_key":"github/sdaps/sda
> ps","title":"sdaps/sdaps","subtitle":"GitHub
> repository","main_image_url":"https://cloud.githubusercontent.com/ass
> ets/143418/17495839/a5054eac-5d88-11e6-95fc-
> 7290892c7bb5.png","avatar_image_url":"https://cloud.githubusercontent
> .com/assets/143418/15842166/7c72db34-2c0b-11e6-9aed-
> b52498112777.png","action":{"name":"Open in GitHub","url":"https://gi
> thub.com/sdaps/sdaps"}},"updates":{"snippets":[{"icon":"PERSON","mess
> age":"@dikey94 in #129: The right alignment + unnecessary 1.2 and 1.2
> headers"}],"action":{"name":"View Issue","url":"https://github.com/sd
> aps/sdaps/issues/129#issuecomment-375427265"}}}

**Comment by dikey94** [comment]
Sure, works.
What's the best way to provide own translations? I'm using stable version

**Comment by benzea** [comment]
Oh, interesting question, as I haven't been backporting all the translation updates.

If you are not afraid of using git, then a good way would be to do the translations in weblate. Then do the following:

 1. Grab the git repository
 2. Run ./setup.py build_i18n
 3. Copy the generated .dict files from `build/share/sdaps/tex/` to `/usr/share/sdaps/tex` or add your dictionary file by passing it via "sdaps X setup_tex --add path/to/file.dict"

**Comment by benzea** [comment]
You can obviously also create a dictionary file like that manually. That shouldn't be hard, but I would appreciate if you copy the information into weblate at some point.

**Comment by dikey94** [comment]
I didn't noticed you have made a release :+1: 
I'd like to share with a community my polish .dict file (don't how enough time to translate everything now).
How about that?

**Comment by benzea** [comment]
Right, polish isn't there yet. If you attach the file here I can put the stings in right away, safes a bit of work.

**Comment by dikey94** [comment]
[polish+example.zip](https://github.com/sdaps/sdaps/files/1849061/polish.example.zip)
Here it is.

Unfortunately I get error:

> babel 2016/02/24 3.9q The Babel package
> ! Package babel Error: You haven't specified a language option.
> See the babel package documentation for explanation.
> Type  H <return>  for immediate help.

running `sdaps ./dk25 setup_tex example.tex --add ./translator-sdaps-dictionary-Polish.dict `

What's the command to select different language?

Cheers.

**Comment by benzea** [comment]
Hey, any chance of getting reasonable information to attribute the commit to you?

**Comment by dikey94** [comment]
Oh sorry. Sure. Feel free to use my real name: Damian Kulig. My e-mail address is dp.kulig@student.uj.edu.pl.
Thanks.

**Comment by benzea** [comment]
Alright, added. One can now translate polish on weblate:
  https://hosted.weblate.org/projects/sdaps/tex/pl/
and
  https://hosted.weblate.org/projects/sdaps/master/pl/
